Makefile

```diff
@@ -1,7 +1,8 @@
+
 BIN_NAME="terraform-provider-instaclustr"
 VERSION=v1.8.0
```

Comment:
It appears that there may be some material change for users of the terraform provider (example - the import looks like it has been updated), could we bump up the version?

Reply:
good point, will bump